snowpack/src/build/optimize.ts

```diff
@@ -36,16 +36,27 @@ const FAKE_BUILD_DIRECTORY = path.join(process.cwd(), '~~bundle~~');
 const FAKE_BUILD_DIRECTORY_REGEX = /.*\~\~bundle\~\~[\\\/]/;
 
 /** Collect deep imports in the given set, recursively. */
-function collectDeepImports(url: string, manifest: SnowpackMetaManifest, set: Set<string>): void {
-  if (set.has(url)) {
+function collectDeepImports(
+  config: SnowpackConfig,
+  url: string,
+  manifest: SnowpackMetaManifest,
+  set: Set<string>,
+): void {
+  const buildPrefix = removeLeadingSlash(config.buildOptions.out.replace(process.cwd(), ''));
+  const normalizedUrl = !url.startsWith(buildPrefix) ? path.join(buildPrefix, url) : url;
```

Comment:
Is this a URL or a local path (should this be `path.posix.join` instead)?

Reply:
This is a local path so I think `path.posix.join` is the answer. Will update now and push up to see what windows has to say about it!